handbook/working_and_growing_here.md

```diff
@@ -180,12 +202,12 @@ We ask that employees be in their role for **at least** 12 months before applyin
 #### Step 1: Explore your interests
 
 1. Look at the [internal career site](https://app5.greenhouse.io/internal_job_board/) to see which roles you might be interested in.
-2. Reach out to your [People Business Partner](https://www.notion.so/People-Business-Partners-7c4b9fca56e245e684bd8d8e79f65910) to confirm eligibility for internal mobility.
+2. Reach out to your [People Business Partner](https://www.notion.so/dbtlabs/Find-your-PBP-f54e1de2eefa413bad7eb251a3a7c19f?pvs=4) to confirm eligibility for internal mobility.
 3. Reach out to the [recruiter](https://www.notion.so/Business-Partner-Coverage-Model-0a4ee30503464b3bb101de1d4c3ac5e8) and hiring manager to have an introductory call:
```

Comment:
Suggest: recruiter -> Talent Acquisition Partner

Reply:
Do you want all references to recruiter updated to Talent Acquisition Partner? I went ahead and did that but can change it back.